src/components/widget.js

```diff
@@ -221,10 +233,38 @@ class Widget extends LitElement {
               <div
                 class="flex flex-row flex-wrap gap-1 justify-start items-center py-1"
               >
-                ${this.member.identities.map(
-                  (identity) =>
-                    html`<obe-identity .identity=${identity}></obe-identity>`
-                )}
+                ${this.member.identities.map((identity, index) => {
+                  // Do not render identities that are above identity limit, unless we are showing all
+                  if (!this.showAllIdentities && index > IDENTITY_LIMIT) {
+                    return;
+                  }
+
+                  // If we have reached limit, show button to show all identities
+                  if (!this.showAllIdentities && index === IDENTITY_LIMIT) {
+                    return html`<button
+                      @click="${this._toggleIdentities}"
+                      class="text-gray-500 cursor-pointer"
+                    >
+                      Show ${this.member.identities.length - IDENTITY_LIMIT}
+                      more linked profiles
+                    </button>`;
+                  }
+
+                  // Otherwise, render identity
+                  return html`<obe-identity
+                    .identity=${identity}
+                  ></obe-identity>`;
+                })}
+
+                <!-- If identities are expanded, show option to hide extras -->
+                ${this.showAllIdentities
+                  ? html`<button
+                      @click="${() => this._toggleIdentities(false)}"
+                      class="text-gray-500 cursor-pointer"
+                    >
+                      Show fewer
```

Comment:
fun fact I’ve learned the difference between less and fewer thanks to this cartoon:

![image](https://github.com/orbit-love/orbit-browser-extension/assets/2587348/0e01faf2-6d1b-48b9-824a-b6b8054aa12d)

Reply:
AFAIK if it's countable, it's fewer. Otherwise, it's less. IE you can have "fewer bread rolls" but "less bread" - my manager at Plymouth Software was **definitely** one of the pedants in your comic :P

Reply:
Oh that makes sense! Thanks!